drivers/flash/soc_flash_nrf.c

```diff
@@ -340,10 +337,15 @@ static int work_in_time_slice(struct flash_op_desc *p_flash_op_desc)
 			   ticker_id, /* flash ticker id */
 			   ticker_ticks_now_get(), /* current tick */
 			   0, /* first int. immediately */
-			   HAL_TICKER_US_TO_TICKS(FLASH_INTERVAL), /* period */
-			   HAL_TICKER_REMAINDER(FLASH_INTERVAL), /* per. rem.*/
+			   /* period */
+			   HAL_TICKER_US_TO_TICKS(
+			   	p_flash_op_desc->context->interval),
```

Comment:
Use local variable instead of double deferencing (3 times in the following lines).